test/pkg/environment/aws/environment.go

```diff
@@ -85,27 +86,31 @@ type ZoneInfo struct {
 
 func NewEnvironment(t *testing.T) *Environment {
 	env := common.NewEnvironment(t)
-	session := session.Must(session.NewSessionWithOptions(
-		session.Options{
-			Config: *request.WithRetryer(
-				&aws.Config{STSRegionalEndpoint: endpoints.RegionalSTSEndpoint},
-				client.DefaultRetryer{NumMaxRetries: 10},
-			),
-			SharedConfigState: session.SharedConfigEnable,
-		},
-	))
+
+	cfg, err := config.LoadDefaultConfig(context.TODO(),
+		config.WithRegion(os.Getenv("AWS_REGION")),
+		config.WithSTSRegionalEndpoint(endpoints.RegionalSTSEndpoint),
+		config.WithRetryer(func() aws.Retryer {
```

Comment:
I don't think we need to do anything custom with the retry here IMO -- I think you're probably fine to just drop it here